subst/react-native.js.flow

```diff
@@ -37,6 +40,44 @@ export type {
   SyntheticEvent as NativeSyntheticEvent,
 } from 'react-native/Libraries/Types/CoreEventTypes';
 
+// In real RN, this takes type params, so you can give a list of valid event
+// names it should know about:
+//
+//   class EmitterSubscription<EventDefinitions: {...}, K: $Keys<EventDefinitions>>
+//     extends _EventSubscription<EventDefinitions, K>
+//     implements EventSubscription {
+//
+// Not so in @types/react-native:
+//
+//   interface EmitterSubscription extends EventSubscription {
+//
+// It just uses `string` for an event name. So, match that unspecificity by
+// choosing unspecific type params.
+//
+// Separately: Not sure why @types/react-native has this as an `interface`
+// with a `new (…)` signature, when real RN has it as a `class`. Seems like
+// `class` is more correct; use that. For contrast, see
+// {Native,}EventEmitter, where @types/r-n uses `class`.
+import _EmitterSubscription from 'react-native/Libraries/vendor/emitter/_EmitterSubscription';
+declare export class EmitterSubscription
+  extends _EmitterSubscription<{| +[string]: mixed |}, string> {}
+
+// Same comment as on EmitterSubscription about an event-names type param.
+//
+// Unlike with EmitterSubscription, @types/react-native follows real RN and
+// makes this a `class`, not `interface`. Not sure why; I guess it's better.
+import _EventEmitter from 'react-native/Libraries/vendor/emitter/_EventEmitter';
+declare export class EventEmitter
+  extends _EventEmitter<{| +[string]: mixed |}> {}
+
+// Same comment as on EmitterSubscription about an event-names type param.
+//
+// Unlike with EmitterSubscription, @types/react-native follows real RN and
+// makes this a `class`, not `interface`. Not sure why; I guess it's better.
+import _NativeEventEmitter from 'react-native/Libraries/EventEmitter/NativeEventEmitter';
+declare export class NativeEventEmitter
+  extends _NativeEventEmitter<{| +[string]: mixed |}> {}
```

Comment:
Hmm, one consequence of this definition is that `NativeEventEmitter` won't be a subclass of `EventEmitter`, while it is in `@types/react-native`.

I guess in the real implementation it isn't, though:
```js
export default class NativeEventEmitter<TEventToArgsMap: {...}>
  implements IEventEmitter<TEventToArgsMap> {
```
so :shrug: hopefully no TS-based library tries to depend on that.